proof/invariant-abstract/IpcDet_AI.thy

```diff
@@ -1096,7 +1270,8 @@ lemma ri_invs[wp]:
   "\<lbrace>\<lambda>s. invs s \<and> st_tcb_at active thread s
                \<and> ex_nonz_cap_to thread s
                \<and> (\<forall>r\<in>zobj_refs ep_cap. ex_nonz_cap_to r s)
-               \<and> (\<forall>r\<in>zobj_refs reply. ex_nonz_cap_to r s)\<rbrace>
+               \<and> (\<forall>r\<in>zobj_refs reply. ex_nonz_cap_to r s)
+               \<and> thread = cur_thread s\<rbrace>
```

Comment:
Since this lemma is `[wp]`: are we sure `thread = cur_thread s` is the only situation in which this function is used?